test/jasmine/tests/sunburst_test.js

```diff
@@ -2062,31 +2062,103 @@ describe('sunburst uniformtext', function() {
         })
         .then(assertTextSizes('using minsize: 9', {
             fontsizes: [12, 12, 12, 12, 12, 12, 12, 12, 12, 12],
-            scales: [0.58, 0.58, 0.58, 0.58, 0.58, 0.58, 0.58, 0.58, 0.58, 0],
+            scales: [1, 1, 1, 1, 1, 1, 1, 1, 1, 0],
         }))
         .then(function() {
             fig.layout.uniformtext.minsize = 32; // set a minsize greater than trace font size
             return Plotly.react(gd, fig);
         })
         .then(assertTextSizes('using minsize: 32', {
             fontsizes: [32, 32, 32, 32, 32, 32, 32, 32, 32, 32],
-            scales: [0, 0.22, 0.22, 0.22, 0.22, 0.22, 0, 0, 0, 0],
+            scales: [0, 1, 1, 1, 1, 1, 0, 0, 0, 0],
         }))
         .then(function() {
             fig.layout.uniformtext.minsize = 16; // set a minsize greater than trace font size
             return Plotly.react(gd, fig);
         })
         .then(assertTextSizes('using minsize: 16', {
             fontsizes: [16, 16, 16, 16, 16, 16, 16, 16, 16, 16],
-            scales: [0.44, 0.44, 0.44, 0.44, 0.44, 0.44, 0.44, 0.44, 0.44, 0],
+            scales: [1, 1, 1, 1, 1, 1, 1, 1, 1, 0],
         }))
         .then(function() {
             fig.layout.uniformtext.mode = 'show';
             return Plotly.react(gd, fig);
         })
         .then(assertTextSizes('using mode: "show"', {
             fontsizes: [16, 16, 16, 16, 16, 16, 16, 16, 16, 16],
-            scales: [0.44, 0.44, 0.44, 0.44, 0.44, 0.44, 0.44, 0.44, 0.44, 0.44],
+            scales: [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+        }))
+        .then(function() {
+            fig.layout.uniformtext = undefined; // back to default
+            return Plotly.react(gd, fig);
+        })
+        .then(assertTextSizes('clear uniformtext', {
+            fontsizes: [12, 12, 12, 12, 12, 12, 12, 12, 12, 12],
+            scales: [1, 1, 1, 1, 1, 1, 1, 1, 1, 0.58],
+        }))
+        .catch(failTest)
+        .then(done);
+    });
+
+    it('should uniform text scales after transition', function(done) {
```

Comment:
I'm getting failures on my laptop:

```
Chrome 78.0.3904 (Linux 0.0.0) sunburst uniformtext should uniform text scales after transition FAILED
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 0', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 1', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 7', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 9', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0, 1, 'scale for element 10', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 11', 'before click'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 0', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 1', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 7', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 9', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0, 1, 'scale for element 10', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
	Error: Expected 0.86 to be close to 0.9725498243340049, 1, 'scale for element 11', 'after click parent'.
	    at <Jasmine>
	    at /tmp/test/jasmine/tests/sunburst_test.js:2012:1 <- /tmp/51a75f08cee2dc3a8cecd7565be9a45d.browserify.js:226894:40
```

Reply:
It should be improved in https://github.com/plotly/plotly.js/pull/4444/commits/2cf7defe563cbdaac46d880e6e54aa6f21965888.

Reply:
Now passing :1st_place_medal: